searchWorker.js

```diff
@@ -1,27 +1,71 @@
+// Access indexed data structure to be used for searching through the documentation
 const req = new XMLHttpRequest();
-req.open('GET', 'decl.bmp', false /* blocking */);
-req.responseType = 'text';
-req.send();
-
-const declNames = req.responseText.split('\n');
 
-// Adapted from the default tokenizer and
-// https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5Cp%7BZ%7D&abb=on&c=on&esc=on&g=&i=
-const SEPARATOR = /[._\n\r \u00A0\u1680\u2000-\u200A\u2028\u2029\u202F\u205F\u3000]+/u
+req.open('GET', 'searchable_data.json', false /* blocking */);
```

Comment:
This file is 7-8x larger than the `decl.bmp` we currently load- I worry this will substantially increase load time for every search.